### PR TITLE
User Vetting/Account Upgrade Docs updates

### PR DESCRIPTION
**Pull request by Whatthefoxsays**
### Checklist
**Required**
- [x] I acknowledge that all my contributions will be made under the project's license.

### PR Details
**Description of the change**:
**Reason for the change**:
**Link to original source**:

Closes #

### CODE REVIEW

File: content/docs/ui/account-and-settings/upgrading-your-account.md [hunk 1]
```diff
@@ -64,3 +62,21 @@ Send to up to 2,000 contacts and 6,000 Emails per month and test all the feature
 Take control of your sending reputation and deliverability with a dedicated IP address, provided with every Pro, Premier and Advanced plan. [Learn More](https://sendgrid.com/blog/shared-and-dedicated-ips-which-should-you-choose/).
 
 </call-out>
+
+### Choose a plan
+
+*Email API*
+
+1. Decide the plan you upgrade to by comparing your current contact storage usage and email sends to the available plans [here](https://sendgrid.com/pricing).
```
Comment (lweinig) [inline content/docs/ui/account-and-settings/upgrading-your-account.md:70]: Remove Contact storage reference: 

Decide the plan you upgrade to by reviewing features and comparing your current email send volume to the available plans [here](https://sendgrid.com/pricing).

File: content/docs/ui/account-and-settings/upgrading-your-account.md [hunk 2]
```diff
@@ -15,43 +15,41 @@ Whether you’re a startup or a large enterprise, SendGrid has several plans tai
 
 ## Upgrade your SendGrid plan
 
-### Choose a plan
+Before you can fully use your upgraded account, you need to complete some verification steps for security purposes.
```
Comment (lweinig) [inline content/docs/ui/account-and-settings/upgrading-your-account.md:18]: Before you can upgrade your account, you need to complete some verification steps for security purposes.

File: content/docs/ui/account-and-settings/upgrading-your-account.md [hunk 3]
```diff
@@ -15,43 +15,41 @@ Whether you’re a startup or a large enterprise, SendGrid has several plans tai
...
-1. Navigate to your [**Account Details**](https://app.sendgrid.com/account/products) page, and then click on the **Your Products** tab.
-1. Under **Email API**, select **Change Plan**.
-1. Find the plan you would like to upgrade to and then click **Select**.
-1. Add your payment information in the popup window and then click **Submit**.
+If you signed up for a Twilio SendGrid account prior to 9/17/2020, your upgrade steps and verification process will look different from what's described below. 
 
+</call-out>
 
-*Marketing Campaigns*
+*To unlock your Twilio SendGrid account:*
```
Comment (lweinig) [inline content/docs/ui/account-and-settings/upgrading-your-account.md:26]: To upgrade your twilio SendGrid account

File: content/docs/ui/account-and-settings/upgrading-your-account.md (hunk 1 above)
Comment (lweinig) [reply content/docs/ui/account-and-settings/upgrading-your-account.md:70]: I think we need a callout somewhere in this section reminding them they will not have access to upgrade until they complete the steps outlined above.

File: content/docs/ui/account-and-settings/verifying-your-account.md [hunk 4]
```diff
@@ -11,42 +11,43 @@ navigation:
   show: true
 ---
 
-When you sign up for a SendGrid account, there are a few steps that our users must complete before they can start using their accounts fully. Even in a pre-verified state, you can send up to 100 emails per day with your SendGrid account. Once you complete the steps presented, your credit usage limits will be unlocked.
+When you sign up for a SendGrid account, there are a few steps that our users must complete before they can start using their accounts fully. Once you complete the steps presented, your credit usage limits will be unlocked.
 
-You will have either one, two, or three separate steps to complete before you fully unlock the credit limits available to your chosen plan level. We require these steps for your security, as well as to help you get started more quickly!
+Depending on whether you choose a *free* or *paid* account, you will have different steps to complete in order to unlock the full potential of your account. We require these steps for your security, as well as to help you get started more quickly!
```
Comment (lweinig) [inline content/docs/ui/account-and-settings/verifying-your-account.md:16]: Everyone starts on a free account so I think the first sentence can be deleted.

File: content/docs/ui/account-and-settings/verifying-your-account.md [hunk 5]
```diff
@@ -11,42 +11,43 @@ navigation:
...
+The time it takes to completely unlock your account can vary based on how quickly you complete the required steps. The faster you complete the steps, the faster you can start sending!
 
-</call-out>
+### Unlock your Free Account
 
-## Verifying your Account
+In order to send email with your free Twilio SendGrid account, you must create and verify a Single Sender. Adding a Sender Identity will help protect your sending reputation and legitimize your account in the eyes of the inbox providers for better deliverability. For more information, see [Single Sender Verification]({{root_url}}/ui/sending-email/sender-verification/).
 
-The time it takes to completely unlock your account can vary based on the number of steps you are required to complete. The faster you complete the steps, the faster you can start sending!
+### Unlock your Paid Account
```
Comment (lweinig) [inline content/docs/ui/account-and-settings/verifying-your-account.md:26]: update title to Unlock the ability to upgrade? Or Access your Upgrade Options

File: content/docs/ui/account-and-settings/verifying-your-account.md [hunk 6]
```diff
@@ -11,42 +11,43 @@ navigation:
...
-* Set up [two-factor authentication]({{root_url}}/ui/account-and-settings/two-factor-authentication/#setting-up-two-factor-authentication)
-* Set up [domain authentication]({{root_url}}/ui/account-and-settings/how-to-set-up-domain-authentication/)
+![]({{root_url}}/img/<domain-auth-send-to-coworker>.png "Send to a coworker tab")
 
-You may need to complete one or more of these steps to completely verify your account. You cannot skip any of the verification steps, or your account won't be completely unlocked.
+</call-out>
 
-### Resend the verification email
+2. Set up [two-factor authentication]({{root_url}}/ui/account-and-settings/two-factor-authentication/#setting-up-two-factor-authentication). Two-factor authentication allows you to make your SendGrid account more secure by requiring authentication beyond a simple username and password when logging in.
+3. Add a [payment method]({{root_url}}/ui/account-and-settings/billing/#payment-methods). Your card will not be charged until your first bill is due.
```
Comment (lweinig) [inline content/docs/ui/account-and-settings/verifying-your-account.md:40]: I don't think this is accurate bc they have to reenter their credit card, I pinged Scott in the slack channel to confirm.

File: content/docs/ui/account-and-settings/verifying-your-account.md [hunk 7]
```diff
@@ -11,42 +11,43 @@ navigation:
...
 
-Two-factor authentication allows you to make your SendGrid account more secure by requiring authentication beyond a simple username and password when logging in. Learn more about setting up and using [two-factor authentication]({{root_url}}/ui/account-and-settings/two-factor-authentication/#setting-up-two-factor-authentication).
+In some instances our compliance team will need to manually verify your account information before you can start sending. In these cases your account should be unlocked within 72 hours of completing the last account verification step. 
 
-## Setting up Domain Authentication
+</call-out>
 
-Domain authentication shows email providers that SendGrid has your permission to send emails on your behalf. For more information, see [How to set up domain authentication]({{root_url}}/ui/account-and-settings/how-to-set-up-domain-authentication/).
 
 ## Upgrading your account
```
Comment (lweinig) [inline content/docs/ui/account-and-settings/verifying-your-account.md:51]: I think you can remove this section now bc the previous section of this page covers it.

File: content/docs/ui/account-and-settings/verifying-your-account.md (hunk 5 above)
Comment (lweinig) [reply content/docs/ui/account-and-settings/verifying-your-account.md:26]: Upgrading your account

File: content/docs/ui/account-and-settings/upgrading-your-account.md [hunk 8]
```diff
@@ -15,43 +15,41 @@ Whether you’re a startup or a large enterprise, SendGrid has several plans tai
...
 
-*Email API*
+<call-out type="warning">
 
-1. Decide the plan you upgrade to by comparing your current contact storage usage and email sends to the available plans [here](https://sendgrid.com/pricing).
-1. Navigate to your [**Account Details**](https://app.sendgrid.com/account/products) page, and then click on the **Your Products** tab.
-1. Under **Email API**, select **Change Plan**.
-1. Find the plan you would like to upgrade to and then click **Select**.
-1. Add your payment information in the popup window and then click **Submit**.
+If you signed up for a Twilio SendGrid account prior to 9/17/2020, your upgrade steps and verification process will look different from what's described below.
```
Comment (lweinig) [inline content/docs/ui/account-and-settings/upgrading-your-account.md:22]: I think you can update this to say '... 9/17/20, you can skip to Choose Your Plan' and link to that section lower on the page.

**Review by lweinig — COMMENTED** [review]
left a few comments, thanks Carly!

File: content/docs/ui/account-and-settings/upgrading-your-account.md (hunk 3 above)
Comment (seanmateer) [reply content/docs/ui/account-and-settings/upgrading-your-account.md:26]: What's the difference between this and ### Unlock your Paid Account below? I think in general I like the purchase prereq steps before the details on navigating to choose plan details as it reflect the real customer journey.

Comment (Whatthefoxsays) [reply content/docs/ui/account-and-settings/upgrading-your-account.md:26]: @seanmateer That section is on the "Verify your account" page. It's a lot of the same info, but we wanted it available in a few places.

**Review by astuteape — APPROVED** [review]

**Pull request merged**